client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -52,12 +53,15 @@ class RemoveDomainDialog extends Component {
 						components: { strong: <strong /> },
 					}
 				) }
-				{ hasTitanWithUs &&
-					' ' +
-						translate(
+				{ hasTitanWithUs && ' ' + isEnabled( 'titan/phase-2' )
+					? translate(
 							'You also have an active Titan Mail subscription for this domain, and your emails will stop ' +
 								'working if you delete your domain.'
-						) }
+					  )
+					: translate(
+							'You also have an active Email subscription for this domain, and your emails will stop ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'You also have an active Titan Mail subscription for this domain, and your emails will stop working if you delete your domain.' )` *ES Score: 8*